src/core/dropdownmenu/dropdownmenu.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="dropdown is-hoverable">
+  <div class="dropdown-trigger">
+    <button class="button">
+      <span>{{ menuName }}</span>
+      <span class="icon is-small">
+        <i class="fas fa-angle-down"></i>
+      </span>
+    </button>
+  </div>
+  <div class="dropdown-menu" id="dropdown-menu" role="menu">
+    <div class="dropdown-content">
+      <ng-container *ngIf="options; else loading">
```

Comment:
Options is expected here as a falsy value, your type for `options` doesn't depict that and an empty array is truethy.